src/hooks/usePagination.tsx

```diff
@@ -33,7 +33,7 @@ export const usePagination = ({
 
   const hasPagination = pagination?.pageSize < dataSize;
   const firstPage = pageIndex === 0;
-  const lastPage = pagination?.pageSize * (pageIndex + 1) > dataSize;
+  const lastPage = pagination?.pageSize * (pageIndex + 1) >= dataSize;
```

Comment:
would prefer if we can rename to `hasLastPage`

firstPage -> isFirstPage

Reply:
how about `isLastPage` instead? since everything should have a last page anyway

Reply:
yup sure, sounds good

Reply:
done [here](https://github.com/keypom/keypom-airfoil/pull/152/commits/18bf094e9c02b7b196f602ac0af3ef0c34147367)